pom.xml

```diff
@@ -29,7 +29,7 @@
         <module>test-old-libraries</module>
         <module>solver</module>
         <module>test-utils</module>
-        <module>wfc</module> <!-- TODO remove once moved in own repository-->
+        <module>wfc</module>
```

Comment:
@seran why removing the comment?

Reply:
One of the previous commit has this comment next to `<module>e2e-tests/emb-json</module>`. Seems like confusion created by IntelliJ when I was creating the module. When I removed the new module from the `pom.xml` this comment also got removed. I'll fix it.